wavefront/client.go

```diff
@@ -0,0 +1,209 @@
+package wavefront
```

Comment:
I don't think we need a new package. We can just add it under `senders`.

Going forward it'll be `senders.NewClient(...)` instead of direct or proxy sender.